docs/guides/guides.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Guides
+toc: true
+weight: 600
+---
+
+# Guides
+
+Because of Crossplane's standardization on the Kubernetes API, it integrates
+well with many other project. Below is a collection of guides and tutorials that
+demonstrate how to use Crossplane with a variety of tools in the GitOps, service
+mesh, and infrastructure provisioning spaces.
+
+- [Connecting AWS managed services to your Argo CD pipeline with open source
```

Comment:
this AWS/Argo link is broken

Reply:
Fixed :+1:

Reply:
yayyy, now i can click this link and see this lovely article with the intro:

> This article is a guest post from Dan Mangum, a software engineer at Upbound.

🚀